### PR TITLE
Add products_id to $pInfo for other product types

### PR DESCRIPTION
**Pull request by torvista**
equate collect_info to /product and short array syntax for product_music

### CODE REVIEW

**Comment by scottcwilson** [comment]
Documentation: adds fix from https://github.com/zencart/zencart/pull/5258/files to other product types.

File: admin/includes/modules/product_music/collect_info.php [hunk 1]
```diff
@@ -80,38 +83,47 @@
...
-  $record_company_array[] = array(
+  $record_company_array[] = [
     'id' => $record_company['record_company_id'],
-    'text' => $record_company['record_company_name']);
+    'text' => $record_company['record_company_name']
+  ];
 }
 
-$music_genre_array = array(array('id' => '', 'text' => TEXT_NONE));
+$music_genre_array = [['id' => '', 'text' => TEXT_NONE]];
```
Comment (scottcwilson) [inline admin/includes/modules/product_music/collect_info.php:118]: Please put on separate lines for readability as you did on line 102-107.  Also, please add comma after last entry on lines 105,125,98, 89, etc.

**Review by scottcwilson — CHANGES_REQUESTED** [review]

**Comment by scottcwilson** [comment]
A couple of small changes then we'll merge.  Thanks for picking up this issue.

File: admin/includes/modules/product_music/collect_info.php (hunk 1 above)
Comment (torvista) [reply admin/includes/modules/product_music/collect_info.php]: ok

**Comment by scottcwilson** [comment]
ok great can you squash?

**Comment by torvista** [comment]
ok

**Pull request merged**